types/hostedBlockchain.go

```diff
@@ -67,6 +67,14 @@ func (c *HostedBlockchains) GetChainFromBytes(chainHash []byte) HostedBlockchain
 	return res.(HostedBlockchain)
 }
 
+func (c *HostedBlockchains) GetChainFromHexString(chainHash string) HostedBlockchain {
```

Comment:
I'm not quite sure how this is going to work. You will pass a hex string and get a deserialized object with all the chain properties from it?

Reply:
Yes, this is retrieving a HostedBlockchain obj from the HostedChains structure. Meaning that if I host Ethereum, I can retrieve the HostedChains by using the hash.

``` go
type HostedBlockchain struct {
	Hash string
	URL  string
}
```

It simply routes to the non-native node running